### PR TITLE
media-gfx/alembic: add 1.8.4

### PR DESCRIPTION
**Pull request by waebbl**
Closes: https://bugs.gentoo.org/881177
Signed-off-by: Bernd Waibel <waebbl-gentoo@posteo.net>

### CODE REVIEW

File: media-gfx/alembic/alembic-1.8.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# py311 needs imath-3.1.6+, see PR #28265
```
Comment (juippis) [inline media-gfx/alembic/alembic-1.8.4.ebuild:6]: Is this `>=` or `>`?

Comment (waebbl) [reply media-gfx/alembic/alembic-1.8.4.ebuild:6]: `>=` 3.1.6 is the latest, so `>` isn't currently possible. It's meant prospective for future updates.

**Review by juippis — APPROVED** [review]
Do you have blender installed with `+alembic` use flag enabled? Could you show me:
**cat /var/db/pkg/media-gfx/blender-*/NEEDED.ELF.2**

```
CMP: media-gfx/alembic-1.8.3-r2/image with media-gfx/alembic-1.8.4/image
 FILES:-usr/lib64/libAlembic.so.1.8.3
 FILES:+usr/lib64/libAlembic.so.1.8.4
```
just making sure this won't break anything. If blender is linked to **libAlembic.so** or **libAlembic.so.1.8** then it's fine. Let me know.

**Comment by waebbl** [comment]
```
X86_64;/usr/bin/blender-3.3;;;libtbb.so.2,libjemalloc.so.2,libspnav.so.0,liblzo2.so.2,libfftw3.so.3,libosdGPU.so.3.4.4,libosdCPU.so.3.4.4,libopenjp2.so.7,libembree3.so.3,libOpenColorIO.so.2.1,liboslcomp.so.1.12,liboslexec.so.1.12,liboslquery.so.1.12,libOpenImageIO.so.2.4.5,libOpenImageIO_Util.so.2.4.5,libpng16.so.16,libjpeg.so.62,libtiff.so.5,libwebp.so.7,libOpenImageDenoise.so.1,libAlembic.so.1.8,libpotrace.so.0,libOpenCOLLADAStreamWriter.so.1.6,libOpenCOLLADASaxFrameworkLoader.so.1.6,libOpenCOLLADAFramework.so.1.6,libOpenCOLLADABaseUtils.so.1.6,libGeneratedSaxParser.so.1.6,libfreetype.so.6,libGL.so.1,libGLEW.so.2.2,libopenvdb.so.9.1,libavcodec.so.58,libavdevice.so.58,libavformat.so.58,libavutil.so.56,libswscale.so.5,libsndfile.so.1,libopenal.so.1,libpulse.so.0,libSDL2-2.0.so.0,libz.so.1,libzstd.so.1,libgmp.so.10,libboost_locale.so.1.80.0,libpugixml.so.1,libhpdf.so,libIex-3_1.so.30,libOpenEXR-3_1.so.30,libpython3.10.so.1.0,libX11.so.6,libXi.so.6,libXxf86vm.so.1,libXfixes.so.3,libXrender.so.1,libstdc++.so.6,libm.so.6,libc.so.6,ld-linux-x86-64.so.2,libgomp.so.1,libgcc_s.so.1;x86_64
```

It's linked against libAlembic.so.1.8

**Comment by waebbl** [comment]
Thanks for merging!

**Pull request closed without merging**